app/src/lib/swagger.ts

```diff
@@ -2,7 +2,7 @@ import { createSwaggerSpec } from "next-swagger-doc";
 
 export const getApiDocs = async () => {
   const spec = createSwaggerSpec({
-    apiFolder: "src/app/api",
+    apiFolder: "src/app/api/v0", // Point to v0 folder to avoid the openapi.json endpoint itself
```

Comment:
We have some API endpoints that aren't in v0, though! Specifically, most of the NextAuth ones. I'm not sure if this is the right move.